boxoffice/views/order.py

```diff
@@ -52,21 +52,20 @@ def jsonify_assignee(assignee):
 def jsonify_order(data):
     order = data['order']
     line_items = []
-    all_line_items = LineItem.query.filter(LineItem.order == order, LineItem.status == LINE_ITEM_STATUS.CONFIRMED).all()
-    for line_item in all_line_items:
+    for line_item in order.line_items:
```

Comment:
The sort can be achieved more efficiently here with SQL as `LineItem.query.filter(LineItem.order == order).order_by("line_item_seq asc")` instead of sorting it in python below.